packages/store/addon/-private/system/internal-model-map.ts

```diff
@@ -42,6 +44,10 @@ export default class InternalModelMap {
     return this._models.length;
   }
 
+  get modelIdentifiers(): StableRecordIdentifier[] {
```

Comment:
we should probably call this recordIdentifiers. We also only use it in a single place, maybe it's not worth creating a method for it?